README.md

```diff
@@ -32,66 +32,64 @@ Dapr codifies the *best practices* for building microservice applications into o
 
 ## How it works
 
-Dapr injects a side-car container/process to each compute unit. The side-car interacts with event triggers and communicates with the compute unit via standard HTTP or gRPC protocols. This enables Dapr to support all existing and future programming languages without requiring you to import frameworks or libraries.
+`Dapr` injects a side-car container/process to each compute unit. The side-car interacts with event triggers and communicates with the compute unit via standard HTTP or gRPC protocols. This enables Dapr to support all existing and future programming languages without requiring you to import frameworks or libraries.
```

Comment:
Changing the style of Dapr here is not what I would recommend, it should blend in here with other text.